### DISCUSSION TITLE
Multiple drawers in the same page

### DISCUSSION THREAD

**Discussion by rishavs** [saadeghi/daisyui#2278] [Q&A] [answered]
I am trying to set up a multi panel layout, with side panels both on the left and right sides and a central column for the content.
However, no matter wherever I place the 2nd drawer - within the content area of the first drawer, or completely out of the first drawer - I am unable to get it to the 3 column layout I want. 

How would I go about having drawers on both sides of my page?

**Comment by saadeghi** [comment, marked as answer]
There was an issue with nesting drawers and it's now fixed in version `3.7.0`.   
Here's an example with nested drawers. One on the left, one on the right. They will be open at `lg` size. They will be collapsed on mobile:  
https://play.tailwindcss.com/4NfFw4GRUb?size=1162x720

**Reply by rishavs** [reply]
Thank you @saadeghi !
